Rebuild 5.4.5 for GDAL3

seisman opened the pull request:
Checklist
* [X] Used a fork of the feedstock to propose changes
* [X] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

@conda-forge-admin, please rerender

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to re-render for you but ran into some issues, please ping conda-forge/core for further assistance.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to re-render for you but ran into some issues, please ping conda-forge/core for further assistance.

seisman commented:
@ocefpaf @leouieda This PR rebuilds 5.4.5 with gdal3. 

I updated the build number and also added libblas, libcblas and liblapack to the runtime dependency list. This PR will close #77.

ocefpaf commented on recipe/meta.yaml:
```diff
@@ -33,6 +34,9 @@ requirements:
   run:
+    - libblas
+    - libcblas
+    - liblapack
```
These should have run_exports and should not be necessary to list them here. Does it fail without them?

seisman replied:
In #77, macOS build fails because liblapack is not installed/found, so I added liblapack to the list.
```
dyld: Library not loaded: @rpath/liblapack.3.dylib
  Referenced from: /usr/local/miniconda/conda-bld/gmt_1565715990180/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/libgmt.5.dylib
  Reason: image not found
```
